lib/ansible/galaxy/data/apb/playbooks/deprovision.yml.j2

```diff
@@ -1,8 +1,8 @@
-- name: {{ role_name }} playbook to deprovision the application
+- name: "{{ role_name }} playbook to deprovision the application"
   hosts: localhost
   gather_facts: false
   connection: local
   vars:
     apb_action: deprovision
   roles:
-  - role: {{ role_name }}
```

Comment:
This shouldn't be a yamlint issue? When defining a list the `-` can be on the same level as the parent of indented. For example these all represent the same value.

```
var:
- a
- b

var:
  - a
  - b

var:
    - a
    - b
```

Reply:
yamllint has a default, a `.yamllint` file allows overriding for your coding standards.
[.yamllint](https://github.com/ansible/ansible/files/2209363/yamllint.txt)